server/socket.js

```diff
@@ -96,9 +100,9 @@ module.exports = io => {
       const peerId = config.peer_id;
       const iceCandidate = config.ice_candidate;
       console.log(`[${socket.id}] relaying ICE candidate to [${peerId}] ${iceCandidate}`);
-
-      if (peerId in sockets) {
-        sockets[peerId].emit('iceCandidate', { 'peer_id': socket.id, 'ice_candidate': iceCandidate });
+      const sockets = store.getState().sockets;
```

Comment:
why is this not colored like the other variables? like line 114. might not be a big deal but it's bothering me.